src/main/components/ResponseComponent.jsx

```diff
@@ -11,9 +11,12 @@ const ResponseComponent = (props) => {
   return (
     <div>
       <p>{`${checkmark} ' - ' ${status}`}</p>
```

Comment:
Going to get a merge conflict here (from #39) but it’ll change soon anyways, so not too worried about it